### PR TITLE
ISPN-14085 Allow to pass new list of indexed entities in update schema API

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-14085

### CODE REVIEW

File: server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheUpdateIndexSchemaTask.java [hunk 1]
```diff
@@ -25,7 +27,14 @@
  * @since 14.0
  */
 public class CacheUpdateIndexSchemaTask extends AdminServerTask<Void> {
-   private static final Set<String> PARAMETERS = Collections.singleton("name");
+   private static final Set<String> PARAMETERS;
```
Comment (pruivo) [inline server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheUpdateIndexSchemaTask.java:30]: nitpicking: could be simplified:
```suggestion
   private static final Set<String> PARAMETERS = Set.of("name", "indexed-entities");
```

**Review by pruivo — COMMENTED** [review]
LGTM, just have a question.
If a user updates the schema with new entities and in boots a new server; is the new server able to see the new entities?

File: server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheUpdateIndexSchemaTask.java (hunk 1 above)
Comment (fax4ever) [reply server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheUpdateIndexSchemaTask.java:30]: Good suggestion. Going to apply it!

**Comment by fax4ever** [comment]
> If a user updates the schema with new entities and in boots a new server; is the new server able to see the new entities?

The change is not persisted :(

**Comment by fax4ever** [comment]
Closing it => the PR is now https://github.com/infinispan/infinispan/pull/10686

**Pull request closed without merging**